server/entities/User.ts

```diff
@@ -31,6 +34,19 @@ export class User {
     @Column('int', { length: 2 })
     role: UserRole = UserRole.Student;
 
+    @Column('varchar', { nullable: true })
+    defaultSubject: string | null;
+
+    @Column('int', { length: 4, nullable: true })
+    graduationYear: number | null;
```

Comment:
https://github.com/typeorm/typeorm/blob/master/docs/entities.md#column-types-for-mysql--mariadb

This should probably be 'year'